src/keeper/utils.ts

```diff
@@ -54,3 +55,29 @@ export async function getNetworkName(networkId: number): Promise<string> {
             throw new KeeperError(`Network with id ${networkId} not supported.`)
     }
 }
+
+// Wrapper for implementing web3 provider. Needed for OpenGSN
+export interface JsonRpcPayload {
+    jsonrpc: string;
+    method: string;
+    params: any[];
+    id?: string | number;
+}
+
+export interface JsonRpcResponse {
+    jsonrpc: string;
+    id: number;
+    result?: any;
+    error?: string;
+}
+
+export class Web3ProviderWrapper {
+    provider: ethers.providers.JsonRpcProvider
+    constructor(provider: ethers.providers.JsonRpcProvider) {
+        this.provider = provider
+    }
+    send(payload: JsonRpcPayload, callback: (error: Error | null, result?: JsonRpcResponse) => void): void {
+        const id = typeof payload.id === 'string' ? parseInt(payload.id) : payload.id
+        this.provider.send(payload.method, payload.params).then(result => callback(null, {jsonrpc: payload.jsonrpc, id, result}))
+    }
+}
```

Comment:
is this still needed? it doesn't seem to be used anywhere

Reply:
it will be useful when somebody actually wants to use the meta transactions, but currently it's only used in the test